errors_validation.go

```diff
@@ -0,0 +1,47 @@
+package jsonschema
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type validationErrorContext interface {
```

Comment:
how user define extensions provide error context ?

Reply:
I didn't have a chance yet to check out the user defined extensions introduced by you recently.